australian-guide-to-legal-citation.csl

```diff
@@ -195,11 +195,11 @@
   </macro>
   <macro name="title-short">
     <choose>
-      <if type="book legislation webpage thesis motion_picture manuscript" match="any">
-        <text variable="title" font-style="italic" text-case="title" form="short"/>
+      <if type="book legislation webpage thesis motion_picture manuscript legal_case article" match="any">
+        <text variable="title-short" font-style="italic" text-case="title" form="short"/>
```

Comment:
also -- what are you intending ```article``` for here? From Zotero that's never used, but maybe you have a different application in mind?